Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceUpdateExecutor.cpp

```diff
@@ -83,7 +83,7 @@ namespace AzToolsFramework
                 return;
             }
 
-            m_instancesUpdateQueue.emplace_back(&(instance->get()));
+            m_instancesUpdateQueue.emplace_back(&instance->get());
```

Comment:
nit: prefer to keep the parenthesis to avoid bad reference errors if the type changes.